tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -157,19 +159,25 @@ def test_build_cluster_data(self):
             cluster_data = dataproc_operator._build_cluster_data()
             self.assertEqual(cluster_data['clusterName'], CLUSTER_NAME)
             self.assertEqual(cluster_data['projectId'], PROJECT_ID)
-            self.assertEqual(cluster_data['config']['softwareConfig'], {'imageVersion': IMAGE_VERSION})
+            self.assertEqual(cluster_data['config']['softwareConfig'],
+                             {'imageVersion': IMAGE_VERSION})
             self.assertEqual(cluster_data['config']['configBucket'], STORAGE_BUCKET)
-            self.assertEqual(cluster_data['config']['workerConfig']['numInstances'], NUM_WORKERS)
-            self.assertEqual(cluster_data['config']['secondaryWorkerConfig']['numInstances'],
-                             NUM_PREEMPTIBLE_WORKERS)
-            self.assertEqual(cluster_data['config']['gceClusterConfig']['serviceAccountScopes'],
+            self.assertEqual(cluster_data['config']['workerConfig']['numInstances'],
+                             NUM_WORKERS)
+            self.assertEqual(
+                cluster_data['config']['secondaryWorkerConfig']['numInstances'],
+                NUM_PREEMPTIBLE_WORKERS)
+            self.assertEqual(
+                cluster_data['config']['gceClusterConfig']['serviceAccountScopes'],
                 SERVICE_ACCOUNT_SCOPES)
+            self.assertEqual(cluster_data['config']['gceClusterConfig']['internalIpOnly'],
```

Comment:
Can you also add tests to check the conditions when `internal_ip_only` is set without `subnetwork_uri` and see if it raises the exception or not?